File: Chronos.Core/Native/Msg.cs
namespace Chronos.Core.Native;

// https://docs.microsoft.com/en-us/windows/win32/api/winuser/ns-winuser-msg
public struct Msg
{
    public nint Hwnd;
    public uint Message;
    public nint WParam;
    public nint LParam;
    public uint Time;
    public Point Pt;
}

File: Chronos.Core.Windows/Native/Msg.fs
namespace Chronos.Core.Windows.Native


// https://docs.microsoft.com/en-us/windows/win32/api/winuser/ns-winuser-msg
[<Struct>]
type Msg =
    { Hwnd: nativeint
      Message: uint32
      WParam: nativeint
      LParam: nativeint
      Time: uint32
      Pt: Point }
